test/critest.bats

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function teardown() {
+    cleanup_test
+}
+
+@test "run the critest suite" {
+    if [[ -z $RUN_CRITEST ]]; then
+        skip "critest because RUN_CRITEST is not set"
+    fi
+
+    sysctl -w net.ipv4.conf.all.route_localnet=1
+    iptables -t nat -I POSTROUTING -s 127.0.0.1 ! -d 127.0.0.1 -j MASQUERADE
+
+    start_crio
+
+    run critest --parallel $JOBS \
```

Comment:
Maybe, it had better to also support CRI_FOCUS and CRI_SKIP environment variables in critest, and then tester can only run specified cases or ignore error cases for debugging, the arguments look like this --ginkgo.focus="${CRI_FOCUS}" --ginkgo.skip="${CRI_SKIP}".